xml/System.Threading/Interlocked.xml

```diff
@@ -1093,7 +1093,7 @@ If `comparand` and the object in `location1` are equal by reference, then `value
 ## Remarks
 
 > [!IMPORTANT]
-> Beginning with .NET Framework 2.0, the <xref:System.Threading.Interlocked.Exchange%60%601%28%60%600%40%2C%60%600%29> method overload provides a type-safe alternative for reference types. We recommend that you call it instead of this overload.
+> Beginning with .NET Framework 2.0, the <xref:System.Threading.Interlocked.Exchange%60%601%28%60%600%40%2C%60%600%2C%60%600%29> method overload provides a type-safe alternative for reference types. We recommend that you call it instead of this overload.
```

Comment:
This link was working in the original version, but this change breaks it. Could you revert your change?